DXRando/DeusEx/Classes/DXRLamp2.uc

```diff
@@ -1,17 +1,65 @@
 //=============================================================================
-// Lamp2 except Warm White.  Fuck daylight bulbs.
+// Lamp2 except with a warm white light at the top.
+// Fuck daylight bulbs.
 // Localization files overwrite the ItemName,
 // so really force it to not be halogen
 //=============================================================================
 class DXRLamp2 injects #var(prefix)Lamp2;
 
+var DynamicLight lt;
+
+function InitLight()
+{
+    local vector loc;
+
+    if (lt == None) {
+        loc = Location;
+        loc.z += CollisionHeight * 1.35;
+
+        lt = Spawn(class'DynamicLight',,, loc);
```

Comment:
you can't actually move lamps, but I guess it's possible one might end up on top of a moveable crate? anyways theoretically it should set the base so that the light moves with the lamp

```suggestion
        lt = Spawn(class'DynamicLight',,, loc);
        lt.SetBase(self);
```